sources/operator.1.10.0.psmdb-operator.json

```diff
@@ -5,6 +5,12 @@
       "product": "psmdb-operator",
       "matrix": {
         "mongod": {
+          "5.0.7-6": {
```

Comment:
If I'm not mistaking this is dangerous! We will add a new version for the old operator version and anybody having automatic upgrades enabled (and automatic upgrades in the old version were enabled by default) will get automatic upgrade.

Reply:
one more thing It was not tested with 1.10.0 operator at all